.github/workflows/e2e-tests.yml

```diff
@@ -232,9 +236,9 @@ jobs:
 
   visual-regression-tests:
     runs-on: ubuntu-20.04
-    timeout-minutes: 30
-    needs: build
-    name: percy-tests-sanity-check-${{ matrix.edition }}
+    timeout-minutes: 45
+    needs: build-ee
```

Comment:
I realized Percy waits for both OSS and EE builds but it only runs against EE uberjar.